Pixel.py

```diff
@@ -124,15 +319,62 @@ def getGreen(self):
     def getBlue(self):
         return self.color[2]
 
-    def distance(self, othercolor):
```

Comment:
We may need to check that othercolor passed in is a type that can be parsed in this way